sof-mach-driver-defconfig

```diff
@@ -1,4 +1,5 @@
 CONFIG_SND_SOC_SOF_HDA_LINK=y
+CONFIG_SND_SOC_SOF_HDA_DISABLE_DMI_L1=y
```

Comment:
is this really needed? I thought it's not recommended?

Reply:
I was surprised to see this as well. Let me send a PR to remove it

Reply:
oops sorry. its my messed up local branch!